day_two_guide/topics/adding_master_host.adoc

```diff
@@ -0,0 +1,177 @@
+=== Adding Masters to *{rhocp}* Environment
+The *{rhocp}* environment can be scaled up horizontally. Masters can be scaled as well regardless if
```

Comment:
'as well regardless' is awkward and probably not even necessary.  'Master can be scaled if `etcd`...'